lib/features/auto-sync-subscriptions.js

```diff
@@ -0,0 +1,24 @@
+const syncChannelName = '$RMQ/sync/my/subscriptions';
+
+module.exports = ({rtm}) =>
+  rtm.subscribe({channel: syncChannelName}).then(() => {
+    return rtm.on(`${syncChannelName}/message`, message => {
+      if (!message || !message.data || !message.data.event) return;
+
+      switch (message.data.event) {
+        case 'subscription-created':
+          rtm.subscribe({channel: message.data.data.channelId});
+          rtm.emit('subscriptionCreated', message.data.data);
+          break;
+        case 'subscription-deleted':
+          rtm.unsubscribe({channel: message.data.data.channelId});
+          rtm.emit('subscriptionDeleted', message.data.data);
+          break;
+        case 'subscription-updated':
+          rtm.emit('subscriptionUpdated', message.data.data);
```

Comment:
In case `message.data.data.allowRead` has changed we need to subscribe/unsubscribe accordingly.
I think for now it is ok to simply act upon the allowRead value.

Reply:
@baranga already mentioned that. Will integrate this :)